struts2-archetype-starter/src/main/resources/archetype-resources/src/main/resources/struts.xml

```diff
@@ -16,7 +16,7 @@
             <result>/WEB-INF/jsp/index.jsp</result>
         </action>
 
-        <action name="helloWorld" class="helloWorldAction">
+        <action name="helloWorld" class="${package}.HelloWorldAction">
```

Comment:
This need to match Spring bean name [here](https://github.com/apache/struts-archetypes/blob/master/struts2-archetype-starter/src/main/resources/archetype-resources/src/main/resources/applicationContext.xml#L6), btw. the package name need to be fixed there

Reply:
Thanks for the review. Sorry I did not realise the `helloWorld` action here looks intentionally different from the `index` action as there is Spring bean for it.

When I setup the project with this archetype from the repositories somehow some files did not generate including `applicationContext.xml`, and have caused this confusion. Now I have tried installing this version here on git again and it works fine. Thanks.

Reply:
Thanks! We fixed a lot of problems and now the archetypes are under test phase, then we release new versions.